docs/tables/datadog_integration_aws.md

```diff
@@ -0,0 +1,20 @@
+# Table: datadog_integration_aws
+
+Datadog AWS integration resource. Integrations allows to bring together all of the metrics and logs from your infrastructure and gain insight into the unified system.
+
+## Examples
```

Comment:
Can you please add 2-3 more examples around showing which filter tags this integration is setup with and showing useful account specific namespace rules info?